### PR TITLE
Add Flasgger + helathcheck for all services

### PR DESCRIPTION
**Pull request by SharonHart**
Decided to use the more lean framework, [flasgger](https://github.com/flasgger/flasgger), over [flask-restplus](https://github.com/noirbizarre/flask-restplus) which has maintainability issues and seems too robust for us. 
There is an option to have the swagger definition in method dcstrings, yaml files, or python dictionaries, will decide on which once we'll document the more heavy methods, but notice how even healthcheck is already pretty long.
By default, each service has a /apidocs endpoint by default showing swagger UI:
image

### CODE REVIEW

**Comment by omri374** [comment]
Please add flasgger to the NOTICE file

File: presidio-analyzer/Pipfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ tldextract = "*"
 numpy = "==1.19.3"
 flask = "==1.1.2"
 pyyaml = "*"
+flasgger = "*"
```
Comment (omri374) [inline presidio-analyzer/Pipfile:13]: Should we restrict the version? How robust is this package in terms of backward compatibility?

Comment (SharonHart) [reply presidio-analyzer/Pipfile:13]: let's restrict, don't think they had breaking changes yet

Comment (SharonHart) [reply presidio-analyzer/Pipfile]: Done

**Review by omri374 — APPROVED** [review]

File: presidio-image-anonymizer/app.py [hunk 2]
```diff
@@ -20,27 +23,38 @@ class Server:
...
+            responses:
+              200:
+                description: OK
+                content:
+                  text/plain:
+                    schema:
+                      type: string
+                      example: ok
+            """
             return "ok"
```
Comment (navalev) [inline presidio-image-anonymizer/app.py:45]: if we're changing the healthcheck, lets return a nicer string. "Presidio Analyzer/Anonymizer/Image-Reducter Service is Up"

File: presidio-image-anonymizer/app.py [hunk 3]
```diff
@@ -4,12 +4,15 @@
...
 from presidio_image_anonymizer.entities import InvalidParamException
-from presidio_image_anonymizer.entities.api_request_convertor import get_json_data, \
-    image_to_byte_array, color_fill_string_to_value
+from presidio_image_anonymizer.entities.api_request_convertor import (
+    get_json_data,
+    image_to_byte_array,
+    color_fill_string_to_value,
+)
 
 DEFAULT_PORT = "3000"
```
Comment (navalev) [inline presidio-image-anonymizer/app.py:18]: add a WELCOME_MESSAGE (same as in the other services)

**Review by navalev — APPROVED** [review]
looks good, added 2 minor comments. As discussed, for the rest of the APIs, the swagger spec will be in external yml files do minimize the noise in app.py

File: presidio-image-anonymizer/app.py (hunk 2 above)
Comment (SharonHart) [reply presidio-image-anonymizer/app.py]: Done, currently leaving 'image anonymizer', we'll do the change in batch

**Review by navalev — APPROVED** [review]

**Pull request merged**